pcdsdevices/attenuator.py

```diff
@@ -1101,6 +1102,101 @@ class AT2L0(FltMvInterface, PVPositionerPC, LightpathInOutMixin):
     num_in = Cpt(InternalSignal, kind='hinted')
     num_out = Cpt(InternalSignal, kind='hinted')
 
```

Comment:
This isn't something you could have known ahead of time, but if you want a new function to pass through the tab completion filter in `hutch-python` you should add something like this near the top of the class:
```
tab_whitelist = ['clear_errors']
```